File: app/src/main/java/com/dupleit/demo/kotlintest/Network/APIService.java
package com.dupleit.demo.kotlintest.Network;

import com.dupleit.demo.kotlintest.SignupModel;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.POST;

interface APIService {
    @FormUrlEncoded
    @POST("checkUserSignup_request")
    Call<SignupModel> checkEmails(@Field("USER_EMAIL") String email, @Field("USER_PASSWORD") String password, @Field("USER_NAME") String name, @Field("USER_MOBILE") String userMobile);


}

File: app/src/main/java/com/dupleit/demo/kotlintest/Network/ApiService.kt
package com.dupleit.demo.kotlintest.Network

import com.dupleit.demo.kotlintest.SignupModel
import retrofit2.Call
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST

/**
 * Created by mandeep on 9/11/17.
 */
interface ApiService {

    @FormUrlEncoded
    @POST("checkUserSignup_request")
    abstract fun checkEmails(@Field("USER_EMAIL") email: String, @Field("USER_PASSWORD") password: String, @Field("USER_NAME") name: String, @Field("USER_MOBILE") userMobile: String): Call<SignupModel>

}